components/PyTorch/Create_PyTorch_Model_Archive/component.yaml

```diff
@@ -0,0 +1,39 @@
+name: Create PyTorch Model Archive
+inputs:
+- {name: Model, type: PyTorchScriptModule}
+- {name: Model name, type: String, default: model}
+- {name: Model version, type: String, default: "1.0"}
+- {name: Handler, type: PythonCode, description: "See https://github.com/pytorch/serve/blob/master/docs/custom_service.md"}
+outputs:
+- {name: Model archive, type: PyTorchModelArchive}
+metadata:
+  annotations:
+    author: Alexey Volkov <alexey.volkov@ark-kun.com>
+implementation:
+  container:
+    image: pytorch/torchserve:0.3.0-cpu
```

Comment:
small suggestion: update to `pytorch/torchserve:0.4.0-cpu`, which is now out.

Reply:
Thank you for the suggestion.
I'll upgrade to the new image when adding a PyTorch serving component, so that I can verify the compatibility.